WireUI/Sources/WireSidebarUI/Views/SidebarView.swift

```diff
@@ -144,18 +145,21 @@ public struct SidebarView<AccountImageView: View, LegalHoldIndicatorView: View>:
 
     private func nonselectableMenuItem(_ menuItem: SidebarNonselectableMenuItem) -> some View {
         let text: Text
+        let accessibilityText: Text
         let icon: String
         let isLink: Bool
         let action: () -> Void
         switch menuItem {
         case .connect:
             text = Text("sidebar.contacts.connect.title", bundle: .module)
+            accessibilityText = Text("sidebar.contacts.connect.title", bundle: .module)
             icon = "person.badge.plus"
             isLink = false
             action = connectAction
 
         case .support:
             text = Text("sidebar.support.title", bundle: .module)
+            accessibilityText = Text("sidebar.support.description", tableName: "Accessibility", bundle: .module)
```

Comment:
Why in some cases we use a tableName and other er don't?

Reply:
because we need different accessibility strings only for some elements, the rest should use the same as in UI